src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/Cells/HealthCertificate/HealthCertificateCell.swift

```diff
@@ -103,6 +103,8 @@ class HealthCertificateCell: UITableViewCell, ReuseIdentifierProviding {
 		return validationButton
 	}()
 	
+	private var _preventCellSelectionInOuterTableView = false
```

Comment:
mmh I guess that's a common naming convention to encapsulate a calc property into a class extension. Because in an extension you can only have a var getter (that could return private var with same _name).
If that's a no go for you, what could be a better name?

```swift
class MyClass {
   private var _some: Bool = true
}

extension MyClass {
   var some: Bool { _some }
}
```

Reply:
Could be renamed to isCellSelectionInOuterTableViewEnabled

Reply:
fine :)